codec-http2/src/test/java/io/netty/handler/codec/http2/CleartextHttp2ServerUpgradeHandlerTest.java

```diff
@@ -155,6 +155,99 @@ public void upgrade() throws Exception {
         assertNull(channel.readOutbound());
     }
 
+    @Test
+    public void upgradeWithMultipleConnectionHeaders() {
+        setUpServerChannel();
+
+        String upgradeString = "GET / HTTP/1.1\r\n" +
+                "Host: example.com\r\n" +
+                "Connection: keep-alive\r\n" +
+                "Connection: Upgrade, HTTP2-Settings\r\n" +
+                "Upgrade: h2c\r\n" +
+                "HTTP2-Settings: AAMAAABkAAQAAP__\r\n\r\n";
+        ByteBuf upgrade = Unpooled.copiedBuffer(upgradeString, CharsetUtil.US_ASCII);
```

Comment:
@arukshani seems like the test code is completely identical except the different `upgradeString`. Consider moving all the code into an extra method and just pass in the `upgradeString` from both of the test methods to remove all the code-duplication

Reply:
Done